api/src/client/ApiFormationClient.test.ts

```diff
@@ -593,11 +600,11 @@ describe("ApiFormationClient", () => {
 
             BusinessInformation: {
               CompanyOrigin: "Foreign",
-              PracticesLaw: false,
               Business: "ForeignForProfitCorporation",
               BusinessName: formationFormData.businessName,
               BusinessDesignator: formationFormData.businessSuffix,
               Naic: userData.profileData.naicsCode,
+              PracticesLaw: "No",
```

Comment:
Same comment as before - shouldn't this be driven by `willPracticeLaw` being set to `false` somewhere in the setup section of this test?

Reply:
Fixed! Thank you.